src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql

```diff
@@ -1,41 +1,184 @@
--- Data for prisons visit schedule
-
--- Start a transaction
+-- This script clears certain tables and re-set auto id's to zero!
```

Comment:
does this file need to be checked in?

Reply:
needs to be checked in.. all prison must be in same script file, cant work otherwise, this is a temporary solution and before we have too many prisons we should move to the admin system.

Reply:
or better - maybe put prison's data configuration scripts and the upload utility in a separate project - within the same repo

Reply:
Yes long term goal all these should be in a separate project UI and server side with orchestrating in-between (prison onboarding/admin system)... should not be in VSiP micro service... but not for now!

Reply:
@AledWEvans @rj-adams but can't we run the script manually on the Prod database once its generated (one time run) so that its not run automatically on deployment. As we might lose existing data due to the truncates?

Reply:
These scripts are un manually they are not part of Fly way : 
scripts in the db.migration.data folder are run by fly way and  scripts in db.migration with naming correct convention "V1_0__name/description.sql" this can allways be checkin the flyway table e.g

```
22	2.10	add permitted session location table	SQL	V2_10__add_permitted_session_location_table.sql	
23		add prison codes	SQL	data/R__add_prison_codes.sql	761666000	visit_scheduler
24		add support	SQL	data/R__add_support.sql	166201964	visit_scheduler
```